src/main/frontend/views/$layout.tsx

```diff
@@ -23,13 +28,16 @@ export default function MainLayout() {
       <div slot="drawer" className="flex flex-col justify-between h-full p-m">
         <header className="flex flex-col gap-m">
           <h1 className="text-l m-0">${currentTitle}</h1>
-          <nav>
+          <SideNav
+            onNavigate={({ path }) => navigate(path!)}
+            location={location}
+          >
             {createMenuItems().map(({ to, icon, title }) => (
-              <NavLink className={navLinkClasses} to={to} key={to}>
+              <SideNavItem path={to} key={to}>
```

Comment:
Can we use an `icon` in this component?

Reply:
Sure: https://vaadin.com/docs/latest/components/side-nav